README.md

```diff
@@ -0,0 +1,31 @@
+# Mapeo Map Server
+
+An in-progress offline map style and tile server for Mapeo
+
+## Technical Details
+
+- [Fastify](https://fastify.io/) for creating the server.
+- [SQLite](https://sqlite.org/index.html) as the underlying persistence layer.
+- [Prisma](https://www.prisma.io/) as a **build** tool for updating the schema, creating migration scripts, and generating schema assets.
+  - Schema diagram can be found [here](/prisma/ERD.svg).
+  - Due to the distributed and local-first nature of Mapeo, migrations are performed at runtime by the server on initialization. See the [migrations implementation](/src/lib/migrations.ts) for more details. It roughly follows the logic that Prisma uses for build-time migrations.
+
+## Features
+
+- [ ] Manages tiles and tilesets (TileJSON)
+
+- [ ] Manages Mapbox map styles
+
+- [ ] Supports importing MBTile files as tilesets
+
+- [ ] Generate styles for raster tilesets
+
+- [ ] Provides info related to downloads and storage
+
+## Developing
+
+Some notes before working on this locally:
+
+- If you make any changes to the schema via `schema.prisma`, run the following commands afterwards:
+  - `npm run prisma:migrate:dev -- --name MIGRATION_NAME_HERE` - creates a new migration in the `prisma/migrations/` directory, which is used by tests and the server if running locally
+  - `npm run updateDataDir` - updates the `data/` directory by copying the `prisma/migrations/` directory, which is used by the server if running locally
```

Comment:
this was under the incorrect assumption that we'd bundle the server to run as some executable or the like in the same data directory as the device. didn't realize that this server would live as is on the file system. can update to assume that the directory structure would be available at runtime

Reply:
Yeah, let's do that. The build script for Mapeo mobile handles copying folders out of node_modules so we can handle it there, and include the prisma/migrations folder in the npm package (it will be included by default).